vtex/faststore
# Fix/ext login texts

## What's the purpose of this pull request?
Only show texts like "Entrar usando Google" or "Escolha uma forma de login" when they make sense/when store providers list exist

## How to test it?
https://preview-97--carrefourbrfood.vtex.app/
https://preview-317--btglobal.vtex.app/
https://preview-410--marinbrasil.vtex.app/
See login page (in marin or boti switch to google/face to see the text)
